### PR TITLE
fix(.net): some TW edits based on feedback

### PR DESCRIPTION
**Pull request by zuluecho9**
Attempt to clear up .net non-IIS install/TS process

### CODE REVIEW

**Review by brnhensley — DISMISSED** [review]
I think these are good improvements @zuluecho9, but I'd like to know if Neil has any input as well. He's out until Monday.

**Comment by brnhensley** [comment]
I think it would also be worth adding something to the [No Data Appears doc](https://docs.newrelic.com/docs/apm/agents/net-agent/troubleshooting/no-data-appears-net/) since that is a major reason they get no data.

**Comment by austin-schaefer** [comment]
Hey @zuluecho9 , is this one merge-able pending TW review?

File: src/content/docs/apm/agents/net-agent/installation/install-net-agent-windows.mdx [hunk 1]
```diff
@@ -183,9 +183,13 @@ Available options are:
 
 ## Enable the agent for apps not in IIS [#enabling-the-agent]
 
-If the application you want to monitor is not hosted in IIS, you will follow the [standard install procedure](#msi) but in addition you must explicitly enable the agent for that application. Enabling the agent differs for .NET Framework and .NET Core applications.
+If the application you want to monitor is not hosted in IIS, you'll follow the [standard install procedure](#msi), with some additional caveats and action items: 
 
-When following the [standard install procedure](#msi), we recommend that you enable **Instrument all**.
+* When following the [standard install procedure](#msi), we recommend you enable **Instrument all**.
+* When you're done with the standard install instructions, you must explicitly enable the agent for that application (explained below). 
+* Because most non-IIS applications are not instrumented automatically, you'll likely need to [set up custom instrumentation](/docs/apm/agents/net-agent/custom-instrumentation/introduction-net-custom-instrumentation) to get data reporting to New Relic.
```
Comment (zuluecho9) [inline src/content/docs/apm/agents/net-agent/installation/install-net-agent-windows.mdx:190]: maybe a link to the compatibility doc. it def is 'most'. some can be self-hosted outside of IIS.

File: src/content/docs/apm/agents/net-agent/installation/install-net-agent-windows.mdx [hunk 2]
```diff
@@ -258,4 +262,10 @@ We recommend setting the environment variables **for each application** you want
   [Here are a few ways you can set this environment variable per process](https://discuss.newrelic.com/t/setting-net-core-agent-environment-variables-per-process/157750), rather than system-wide.
 </Callout>
 
+### .NET Core and custom instrumentation [#custom-instrumentation]
+
+ASP.NET Core apps are automatically instrumented upon installation, but other .NET Core frameworks will require [custom instrumentation](/docs/agents/net-agent/custom-instrumentation/introduction-net-custom-instrumentation)
```
Comment (zuluecho9) [inline src/content/docs/apm/agents/net-agent/installation/install-net-agent-windows.mdx:267]: maybe say 'most', with link to compatibility doc.

Comment (zuluecho9) [reply src/content/docs/apm/agents/net-agent/installation/install-net-agent-windows.mdx:267]: https://docswebsitedevelop-nettroubleshootwindowsinstall.gtsb.io/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core/#app-frameworks

File: src/content/docs/apm/agents/net-agent/troubleshooting/no-data-appears-net.mdx [hunk 3]
```diff
@@ -70,6 +70,16 @@ To troubleshoot the most common reasons for missing data, try these solutions as
       </td>
     </tr>
 
+    <tr>
+      <td>
+        **.NET Core:** May require custom instrumentation.
+      </td>
+
+      <td>
+        ASP.NET Core apps are automatically instrumented after installation, but other .NET Core frameworks require the additional step of [custom instrumentation](/docs/agents/net-agent/custom-instrumentation/introduction-net-custom-instrumentation).
```
Comment (zuluecho9) [inline src/content/docs/apm/agents/net-agent/troubleshooting/no-data-appears-net.mdx:79]: fix this

**Pull request merged**